build-system/tasks/pr-deploy-bot-utils.js

```diff
@@ -16,12 +16,20 @@
 'use strict';
 
 const fs = require('fs-extra');
+const log = require('fancy-log');
 const path = require('path');
 const request = require('request-promise');
+const {cyan, green} = require('ansi-colors');
 const {gitCommitHash} = require('../git');
 const {replaceUrls: replaceUrlsAppUtil} = require('../app-utils');
 const {travisBuildNumber} = require('../travis');
 
+const baseUrl = 'https://amp-pr-deploy-bot.appspot.com/v0/pr-deploy/';
+const results = new Map()
+  .set('success', 0)
+  .set('errored', 1)
+  .set('skipped', 2);
```

Comment:
Q: why not pass the status strings directly to the app, and have the app `switch` on those instead on converting numbers back and forth?